apiserver/facades/client/cloud/instance_information_test.go

```diff
@@ -86,6 +88,7 @@ func (p *instanceTypesSuite) TestInstanceTypes(c *gc.C) {
 	p.pool.EXPECT().GetModelCallContext(coretesting.ModelTag.Id()).Return(p.credcommonPersistentBackend,
 		context.NewEmptyCloudCallContext(), func() bool { return false }, nil)
 	p.backend.EXPECT().ControllerTag().Return(coretesting.ControllerTag)
+	p.backend.EXPECT().ControllerConfig().Return(coretesting.FakeControllerConfig(), nil)
```

Comment:
In another place we just use a controller config map with  a single "controller-uuid" key. Let's do that here too. No need for an entire config.